Core/src/org/sleuthkit/autopsy/examples/SampleDataSourceIngestModule.java

```diff
@@ -56,70 +62,107 @@
  */
 class SampleDataSourceIngestModule implements DataSourceIngestModule {
 
-    private final boolean skipKnownFiles;
     private IngestJobContext context = null;
+    private static final Logger logger = Logger.getLogger(NbBundle.getMessage(SampleDataSourceIngestModuleFactory.class, "SampleDataSourceIngestModuleFactory.moduleName"));
 
     SampleDataSourceIngestModule(SampleModuleIngestJobSettings settings) {
-        this.skipKnownFiles = settings.skipKnownFiles();
     }
```

Comment:
This default constructor should be omitted in favor of the generated default constructor now that it has nothing to do.